PluginJira/Helper/Settings.cs

```diff
@@ -8,6 +8,7 @@ public class Settings
         public string Username { get; set; }
         public string ApiKey { get; set; }
         public string Tenant { get; set; }
+        public string Project { get; set; }
```

Comment:
we should make this a List<string> to allow multiple projects to be selected in one connection